be/src/exec/vectorized/analytor.cpp

```diff
@@ -547,9 +562,12 @@ bool Analytor::find_and_check_peer_group_end(bool is_found_partition_end_genuine
     _found_peer_group_end = _found_partition_end;
```

Comment:
Could the `find_peer_group_end` use the algorithm?

Reply:
Yes, exactly, next pr~